exec/java-exec/src/main/java/org/apache/drill/exec/store/plan/rel/PluginAggregateRel.java

```diff
@@ -51,7 +51,7 @@ public Aggregate copy(RelTraitSet traitSet, RelNode input, ImmutableBitSet group
 
   @Override
   public RelOptCost computeSelfCost(RelOptPlanner planner, RelMetadataQuery mq) {
-    return super.computeSelfCost(planner, mq).multiplyBy(0.1);
+    return super.computeLogicalAggCost(planner, mq).multiplyBy(0.1);
```

Comment:
Are these Plugin*Rel RelNodes used when operations are pushed down to storage plugins?

Reply:
Yes, they represent part of the pushed-down query, so it is possible to find out the most optimal query for both Drill and actual storage where the query is pushed down.